airbyte-integrations/connectors/source-mailchimp/source_mailchimp/schemas/segments.json

```diff
@@ -44,7 +46,7 @@
                 "type": ["null", "string"]
               },
               "value": {
-                "type": ["null", "string"]
+                "type": ["null", "integer", "number", "string"]
```

Comment:
Do we support multiple types in a single destination column? I believe that a destination column can only handle one type. Fields with multiple types need to be standardized via transformation.

Reply:
Thanks for the clarification! I've updated this to be an additional undeclared property for now as our sandbox data is very limited, and have added the type for the nested `conditions/items` in the automations stream as requested.

Reply:
Thanks for the clarification! I've updated this field to be an additional undeclared property for now as we don't have much sandbox data to test against, and I've also added the type for the array items in the automations stream as you requested.